### PR TITLE
add user Form, switch button

### PR DESCRIPTION
**Pull request by DevUnderConstruction**

### CODE REVIEW

File: components/UserForm.js [hunk 1]
```diff
@@ -0,0 +1,105 @@
+export default function UserForm({ client, handleSubmit, handleInputChange }) {
+  return (
+    <form onSubmit={handleSubmit}>
+      <label htmlFor="firstName">Name{client.firstName}</label>
```
Comment (AntjeWalter) [inline components/UserForm.js]: Why is the {client.firstName} in here? At the LastName you don't have that in the label...

File: components/UserForm.js [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+      <label htmlFor="zib">ZIB</label>
+      <input
+        type="text"
+        id="zib"
+        name="zib"
+        //defaultValue={curUser.zib}
+        value={client.zib}
+        onChange={handleInputChange}
+        required
+      />
```
Comment (AntjeWalter) [inline components/UserForm.js]: Do you mean ZIP for "Postleitzahl"? Then it is with a P :) Otherwise, forget this comment!

File: pages/index.js [hunk 3]
```diff
@@ -1,15 +1,59 @@
...
 import Header from "../components/Header";
 import Usercard from "../components/Usercard";
+import Footer from "../components/Footer";
+import UserForm from "../components/UserForm";
+import { useState } from "react";
 
+const curUser = {
+  id: 1,
+  firstName: "Rainer",
+  lastName: "Zufall",
```
Comment (AntjeWalter) [inline pages/index.js:11]: Hahaha :D

**Review by AntjeWalter — COMMENTED** [review]
Just some minor comments :)

File: components/UserForm.js (hunk 1 above)
Comment (DevUnderConstruction) [reply components/UserForm.js]: It was only for testing and i will delete it

File: components/UserForm.js (hunk 2 above)
Comment (DevUnderConstruction) [reply components/UserForm.js]: typo☹️

File: components/UserForm.js [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+
+      <label htmlFor="lastName">LastName</label>
+      <input
+        type="text"
+        id="lastName"
+        name="lastName"
+        //defaultValue={curUser.lastName}
```
Comment (julialoeschel) [inline components/UserForm.js]: please delete unused code for PR ;)

File: pages/index.js [hunk 5]
```diff
@@ -1,15 +1,59 @@
...
+    const { name, value } = event.target;
+    setClient((prevProps) => ({
+      ...prevProps,
+      [name]: value,
+    }));
+  };
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    console.log(client);
```
Comment (julialoeschel) [inline pages/index.js]: no console.logs in PRs ;)  and it is not doing anything.

File: pages/index.js [hunk 6]
```diff
@@ -1,15 +1,59 @@
...
       </Head>
       <Header />
-      <Usercard />
+
+      {nav ? (
+        <Usercard client={client} />
+      ) : (
+        <UserForm
+          client={client}
+          handleSubmit={handleSubmit}
```
Comment (julialoeschel) [inline pages/index.js]: I think you dont need this here, you are listening to your changes in handleInputChange. right now I dont see any use of the submit here.

**Review by julialoeschel — COMMENTED** [review]
good job!

i am exited to see where this is going. 
:octocat:

File: pages/index.js (hunk 5 above)
Comment (DevUnderConstruction) [reply pages/index.js]: fixed

File: pages/index.js (hunk 6 above)
Comment (DevUnderConstruction) [reply pages/index.js]: Without it changes do not appaer in my UserCard

**Pull request merged**